integration/tests/basic/test_rpc_calls.py

```diff
@@ -194,22 +203,47 @@ def test_eth_gas_price(self, param: tp.Union[str, None]):
             (Tag.EARLIEST, None),
         ],
     )
-    def test_eth_get_logs_via_tags(self, tag1: Tag, tag2: Tag, erc20_contract: tp.Any):
-        """Verify implemented rpc calls work eth_getLogs"""
-        contract, deploy_tx, _ = erc20_contract
-        topics = get_event_signatures(contract.abi)
-        params = {
-            "address": deploy_tx.contractAddress,
-            "topics": topics,
-        }
+    @pytest.mark.parametrize(
+        "param_fields", [("address", "topics"), ("address",), ("topics",)]
+    )
+    def test_get_rpc_logs(self, event_caller_contract, param_fields, tag1, tag2):
```

Comment:
it is still  test to check logs via tags

check this page please
https://ethereum.org/en/developers/docs/apis/json-rpc/

fromBlock and toBlock can be tags and QUANTITY value

address, topics and blockhash - optional values
we should check all variants and negative tests too

Reply:
Added tests for fromBlock&toBlock QUANTITY value, blockHash and negative scenarios too.